package.json

```diff
@@ -15,7 +15,8 @@
     "clean": "rm -rf dist",
     "test": "jest --colors --config jest.config.js src",
     "coverage": "jest --colors --config jest.config.js --coverage",
-    "lint": "eslint --report-unused-disable-directives --config .eslintrc.js \"{run_local.js,src,bin,__{tests,mocks}__}/**/*.js\"",
+    "lint:all": "yarn lint \"{run_local.js,src,bin,__{tests,mocks}__}/**/*.js\"",
+    "lint": "eslint --report-unused-disable-directives --config .eslintrc.js --ignore-path .eslintignore",
```

Comment:
Our `lint-staged` was passing files to `lint` but we were still linting everything. So I moved the paths list to a `lint:all` command and left this without them (but added the ignore file).